helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -206,6 +206,14 @@ extraConfigMaps:
       </exclude>
     </filter>
 
+    <filter kubernetes.var.log.containers.ingress-nginx-controller-**>
+      @type grep
+      <exclude>
+        key message
+        pattern /\[error\]\s[0-9]+#[0-9]+:\s\*([A-Za-z0-9]+( [A-Za-z0-9]+)+):/
+      </exclude>
+    </filter>
+
```

Comment:
Could you add another filter to remove any messages containing binary symbols, `\\xXY` XY => 0-9 A-F, from `kubernetes.var.log.containers.ingress-nginx-controller-**`